src/data_collector.py

```diff
@@ -0,0 +1,52 @@
+import aiohttp
+import asyncio
+import boto3
+import json
+
+STATUS_NODES = [
+    'node.explorer.hathor.network',
+    'node1.mainnet.hathor.network',
+    'node2.mainnet.hathor.network'
+]
+
+def process_node_status(node_status: dict):
+    session = boto3.session.Session()
+    lamb = session.client("lambda", endpoint_url="http://localhost:3002")
```

Comment:
What's this endpoint?

Reply:
Maybe this is the local addess when running Serverless to simulate the Lambdas.

I think we could add an optional ENV VAR for this, that would be set only in development environments.

Reply:
There's also [pydantic](https://pypi.org/project/pydantic/), it has a good settings module and works with typing to ensure settings types.

Reply:
Pydantic seems good too. I just think that we should standardize which tools we will use in our projects for those common tasks. The one we choose here should be our standard (unless we already have one).